client/src/assets/js/app/typing-lesson.js

```diff
@@ -103,7 +103,16 @@
       },
       isPassable: function() {
         return this.accuracy >= MIN_PASSING_ACCURACY && this.wpm >= MIN_PASSING_WPM;
-      }
+      },
+      endLesson: function() {
+        this.markCompleted();
+      },
+      accuracyLabel: function() {
+        return (this.accuracy) ? `${this.accuracy}%` : '-';
+      },
+      isRunning: function() {
+        return this.state === LevelStates.Running;
```

Comment:
For clarity, we should wrap conditionals in `()` if we use the result as a value:

`return (this.state === LevelStates.Running);`